nautobot/core/templates/admin/base.html

```diff
@@ -0,0 +1,183 @@
+
+    <style>
+        .description {
+            padding-bottom: 20px;
+        }
+        .help-block {
+            white-space: initial;
+            word-wrap: break-word;
+        }
+        .related-widget-wrapper {
+            float: none;
+        }
+        .selectfilter {
+            width: 100%;
+        }
+        .vLargeTextField, .vXMLLargeTextField {
+            width: 100%;
+        }
+        .vTextField {
+            width: 100%;
+        }
+        .related-widget-wrapper > select {
+            width: 100%;
+        }
+        .form-horizontal .control-label {
+            text-align: left;
+        }
+        input[type="checkbox"] {
+            margin-left: 0 !important;
+        }
+        textarea {
+            width: 100%;
+        }
+    </style>
+</head>
+<body {% if is_popup %}style="padding-top: 0px;"{% endif %}>
+{% if not is_popup %}
+    {% include 'inc/nav_menu.html' %}
+{% endif %}
+<div class="container-fluid wrapper" {% if is_popup %}style="padding-bottom: 0px;"{% endif %}>
+    {% if settings.BANNER_TOP %}
+        <div class="alert alert-info text-center" role="alert">
+            {{ settings.BANNER_TOP|safe }}
+        </div>
+    {% endif %}
+    {% if settings.MAINTENANCE_MODE %}
+        <div class="alert alert-warning text-center" role="alert">
+            <h4><i class="mdi mdi-alert"></i> Maintenance Mode</h4>
+            <p>Nautobot is currently in maintenance mode. Functionality may be limited.</p>
+        </div>
+    {% endif %}
+    {% plugin_banners %}
+    {% for message in messages %}
+        <div class="alert alert-{{ message.tags }} alert-dismissable" role="alert">
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                <span>&times;</span>
+            </button>
+            {{ message }}
+        </div>
+    {% endfor %}
+    <!-- Content -->
+    <div class="container-fluid wrapper">
+        {% if not is_popup %}
+            <div class="row noprint">
+                {% block breadcrumbs %}
+                {% endblock %}
+            </div>
+        {% endif %}
+        <div>
+            <div>
+                {% block pretitle %}{% endblock %}
+                <div class="pull-right noprint">
+                    {% block object-tools %}{% endblock %}
+                </div>
+                <div>
+                    {% comment %} {% block content-navbar-collapse %}
+                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#content-navbar-collapse" aria-expanded="false" aria-controls="navbar">
+                            <span class="sr-only">Toggle navigation</span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                        </button>
+                    {% endblock %} {% endcomment %}
```

Comment:
Intentional that this block is commented out? Is it something that needs to be fixed, or should it just be removed?

Reply:
I was leaving it just incase, but can be removed

Reply:
In case of what? :-) Does it represent functionality that was present in the original templates that we've lost in this conversion?